test/e2e/internalencryption/README.md

```diff
@@ -0,0 +1,15 @@
+# Internal Encryption E2E Tests
+
+In order to test Internal Encryption, this test looks at `tls_connection_count` metrics from the Activator and QueueProxy.
+
+The `metricsreader` test image [here](../../test_images/metricsreader/README.md) was created for this purpose. Given the PodIPs of the Activator and the Knative Service pod (i.e. the QueueProxy), it will make requests to each respective `/metrics` endpoint, pull out the `*_tls_connection_count` metric, and respond with the value.
+
+The test works as follows:
+* The [setup script](../../e2e-internal-encryption-tests.sh) configures the `dataplane-trust` config in `config-network` to `enabled`.
+* The [test](internalencryption_test.go) deploys the `metricsreader` Knative Service. This service uses an annotation to set the min-scale to 1 so that the PodIP for the pod is available.
```

Comment:
You may want to set max-scale as well, to make sure you don't get a split-brain scenario.